src/osip-monitoring-controller/src/main/resources/Bundle.properties

```diff
@@ -19,3 +18,23 @@ monitoring.tank.progress.fillLevel=Fill level
 monitoring.tank.progress.temperature=Temperature
 monitoring.mixTank.motorSpeed=Motor speed
 monitoring.mixTank.color=Color
+monitoring.helpdialog.title=Help
+monitoring.helpdialog.mainTab.header=Shell
+monitoring.helpdialog.mainTab.introduction=The OSIP monitoring shell can be used to monitor the tanks, pipes and alarms of the production site simulation via the OPC UA protocol.
+monitoring.helpdialog.mainTab.layoutSection=Shell Layout
+monitoring.helpdialog.mainTab.layoutText=The layout of the monitoring shell can be divided into five parts. At first there are the information about the upper tanks of the production site. These boxes are located at the upper side of the shell and are bordered in the color of the liquid in the tank. Right of those parts is the information about the mix tank which is also bordered in the color of the mix tank. Underneath those boxes there is a textfield which is used to print alarm messages.
+monitoring.helpdialog.mainTab.alarmSection=Alarms
+monitoring.helpdialog.mainTab.alarmText=Every tank box has four possible alarms which are shown as small green or red circles at the upper left corner of the box. If the circle is green the alarm is not active, if it is red then the alarm is active. Additionally there is a big red and green circle at the bottom right of the shell. If the red circle is filled, some alarm is active and if the green circle is filled, no alarm is active.
+monitoring.helpdialog.mainTab.gaugeSection=Gauges
+monitoring.helpdialog.mainTab.gaugeText=Every tank box has four gauges showing various properties of the tank. The gauge at the upper left side shows the input flow rate of liquid flowing into the tank. The gauge at the upper right side shows the output flow rate of liquid flowing out of the tank. Below is a fill rate gauge and a temperature scale.
+monitoring.helpdialog.mainTab.progressSection=Progress diagram
+monitoring.helpdialog.mainTab.progressText=Below the gauges are two progress diagrams, accessible via two tabs, which show the progress of the fill level and the temperature. This can be used to analize the history of the fill level and temperature changes.
+monitoring.helpdialog.settingsTab.header=Settings
+monitoring.helpdialog.settingsTab.introduction=The settings can be opened by clicking on the "File" menu and selecting the menu entry "Settings"
+monitoring.helpdialog.settingsTab.generalSection=General Settings
+monitoring.helpdialog.settingsTab.generalText=The general settings tab can be used to configure the update interval, which is the frequency used by the monitor to request data from the production site simulation. Additionally one can specify the hostname or IP-address of the production site simulation by filling it into the "Server-Hostname" field. Below this field one can select the IP ports of every tank.
```

Comment:
The general settings tab can be used to configure the update interval **[kein Komma]** which is the frequency used by the **monitoring shell** to request data from the production site simulation. Additionally **,** one can specify the hostname or IP-address of the production site simulation by filling it into the "Server-Hostname" field. Below this field **,** one can select the ports **for** every tank.